tests/common/test_common_constants.py

```diff
@@ -0,0 +1,143 @@
+    + "\nContact your security administrator for READ access before using pyRACF."
+    + "\nReview our documentation at https://ambitus.github.io/pyracf/ as well!\n"
+)
+
+TEST_SETUP_PRECHECK_DEFINED_PROFILE_TEXT = (
+    "IRR.IRRSMO00.PRECHECK is now defined with a `Universal Access` of None."
+    + "\nContact your security administrator for READ access before using pyRACF."
+    + "\nOther users of pyRACF will also need to have at least read access."
+    + "\nYou may also need to REFRESH the `XFACILIT` class."
+    + "\nReview our documentation at https://ambitus.github.io/pyracf/ as well!\n"
+)
+
+# ============================================================================
+# Customize Segment Traits
+# ============================================================================
+
+# Alter User Traits
+TEST_ALTER_USER_CSDATA_AND_OMVS_REQUEST_TRAITS = {
+    "base:special": False,
+    "omvs:home_directory": "/u/clarinet",
+    "omvs:default_shell": False,
+    "csdata:tstcsfld": "testval",
+}
+TEST_ALTER_USER_CSDATA_REQUEST_TRAITS = {
+    "base:special": False,
+    "csdata:tstcsfld": "testval",
+}
+
+# Valid Segment Traits Updates
+TEST_USER_REPLACE_SEGMENT_TRAITS = {
+    "base": {"base:special": "alt:special"},
+    "csdata": {"csdata:tstcsfld": "tstcsfld"},
+}
+
+TEST_USER_ADDITIONAL_SEGMENT_TRAITS = {"csdata": {"csdata:tstcsfld": "tstcsfld"}}
+
+# Alter User Requests
+TEST_ALTER_USER_REQUEST_REPLACE_SEGMENTS_XML = get_sample(
+    "alter_user_request_replace_segments.xml", "user"
+)
+TEST_ALTER_USER_REQUEST_UPDATE_SEGMENTS_XML = get_sample(
+    "alter_user_request_update_segments.xml", "user"
+)
+
+# Extract User Results
+TEST_EXTRACT_USER_RESULT_BASE_OMVS_CSDATA_SUCCESS_XML = get_sample(
+    "extract_user_result_base_omvs_csdata_success.xml", "user"
+)
+TEST_EXTRACT_USER_RESULT_BASE_OMVS_CSDATA_SUCCESS_DICTIONARY = get_sample(
+    "extract_user_result_base_omvs_csdata_success.json", "user"
+)
+
+# ============================================================================
+# Run As UserId
+# ============================================================================
+
+TEST_RUNNING_USERID = "ESWIFT"
+TEST_ALTER_USER_REQUEST_TRAITS = {
+    "base:special": False,
+    "omvs:home_directory": "/u/clarinet",
+    "omvs:default_shell": False,
+}
+TEST_ALTER_USER_REQUEST_XML = get_sample("alter_user_request.xml", "user")
+TEST_EXTRACT_USER_RESULT_BASE_ONLY_SUCCESS_XML = get_sample(
+    "extract_user_result_base_only_success.xml", "user"
+)
+TEST_ALTER_USER_SUCCESS_LOG = get_sample("alter_user_success.log", "user")
+TEST_ALTER_USER_SUCCESS_AS_ESWIFT_LOG = get_sample(
+    "alter_user_success_as_eswift.log", "user"
+)
+TEST_ALTER_USER_RESULT_SUCCESS_XML = get_sample("alter_user_result_success.xml", "user")
+TEST_EXTRACT_RESOURCE_PRECHECK_AS_SQUIDWRD_LOG = get_sample(
+    "extract_resource_precheck_as_squidwrd.log", "resource"
+)
+
+# ============================================================================
+# Null Response Error
```

Comment:
This should reference `DownstreamFatalError` now, no?

Reply:
fixed.